signac/contrib/project.py

```diff
@@ -850,51 +817,47 @@ def _find_job_ids(self, filter=None, doc_filter=None, index=None):
             If the filters are not JSON serializable.
         ValueError
             If the filters are invalid.
-        RuntimeError
-            If the filters are not supported by the index.
 
         Notes
         -----
-        If all arguments are ``None``, this method skips indexing the data
-        space and instead simply iterates over all job directories. This
-        code path can be much faster for certain use cases since it defers
-        all work that would be required to construct an index, so in
-        performance-critical applications where no filtering of the data space
-        is required, passing no arguments to this method (as opposed to empty
-        dict filters) is recommended.
+        If all arguments are ``None``, this method simply returns a list of all
+        job directories. This code path can be much faster for certain use cases
+        since it defers all work that would be required to construct an index,
+        so in performance-critical applications where no filtering of the data
+        space is required, passing no arguments to this method (as opposed to
+        empty dict filters) is recommended.
```

Comment:
I think empty dict filters should be equivalent to no argument due to the condition tested in line 831.

Reply:
Done in 0a490d0a.